eventstore/models.py

```diff
@@ -668,22 +657,23 @@ def has_value(v):
                 self.data[k] = v
 
     def update_post_screening_study_arms(self, risk, source):
-        if self.age == Covid19Triage.AGE_U18:
+        if self.age == Covid19Triage.AGE_U18 and not self.province:
```

Comment:
I'm not sure what the change is supposed to be, just want to double check that this is correct, seems odd that they would be under 18, but some of them have a province and others don't

Reply:
I just added this because the randomisation relies on it being there for all 3 studies. Some of our tests don't specify a province then it fails.

Reply:
Should it not maybe then be `or`, so that if they're under 18 or they don't have a province, then it's skipped?

Reply:
yeah 'or' would be correct